### ISSUE TITLE
Ghostrider

### ISSUE THREAD

**Issue by Ranchdad86** [xmrig/xmrig-proxy#554]
The proxy will not work with ghostrider and I have to use xmrig on each system directly to the pool. Is there a plan to fix this so that the proxy works with ghostrider

**Comment by sapperwerewolf** [comment]
???

**Comment by Ranchdad86** [comment]
> ???

XMPROXY does not work with ghostrider algo. I was asking if there is a plan to add support for ghostrider algo to xmproxy

**Comment by sapperwerewolf** [comment]
Yes, I know it doesn't work. We are waiting for the developer's response on this improvement.

**Comment by atta2022** [comment]
Would be nice to fix this

**Comment by webhostsg** [comment]
up this, I wish it would support ghostrider.

**Comment by Ranchdad86** [comment]
> up this, I wish it would support ghostrider.

The UltimateProxy dev is working on adding it to his which would also support ratio switching between pools.

He is also working to add rx/yada since xmproxy doesn't have support for it yet either

**Comment by webhostsg** [comment]
> UltimateProxy

Ok, great I'll take a look and wait for it to be added. Currently, I'm using [XMRig-Proxy-Frontend](https://github.com/pcca-matrix/XMRig-Proxy-Frontend) for pool switching; it supports ration/time. I switch pool every 1 hour.

**Comment by WhyTea78** [comment]
I would also very much welcome it if ghostrider were supported by xmrig-proxy.
